plugins/destiny.py

```diff
@@ -23,6 +24,8 @@
     'secondsPlayed', 'longestSingleLife', 'orbsDropped', 'precisionKills',
     'precisionRate', 'suicides', 'winRate', 'publicEventsCompleted']
 
+# Reset is at 9am UTC, so -9h UTC makes the reset midnight
+DAILY_RESET_TIMEZONE = datetime.timezone(-datetime.timedelta(hours=9))
```

Comment:
datetime doesn't have a method/function/subclass called timezone. check dir(datetime). you are thinking of the time library. time.timezone. but again, this isn't needed.